### ISSUE TITLE
access88

### ISSUE THREAD

**Issue by Missa88** [Missa88/M#1]
￼
￼￼Submit
Semi-compliant
Your efforts are noticeable, but there's still a way to go to achieve WCAG 2.1 compliance

It's ready! Check your mail for a download link
Looking to fix the results? accessiBe.com
accessiBe Is not properly installed!
We've identified you as an accessiBe customer. However, the installation code is added improperly through a script manager that prevents the load of accessiBe.

Therefore, this audit represents the compliance level of your website WITHOUT accessiBe.

No accessibility system detected
Clickables
SCORE: 75
Good ScoreElements with button functionality should be tagged for assistive technology
19
Total Elements
18
Successes
1
Failures
Requirement: Elements that behave as buttons but are built using other tags such as span, div, a or others, should include a "role" attribute that equals to "button".

Code snapshots of some failed elements
<a href="javascript:void(0)" data-element-location="footer" data-element-type="link" data-element-label="cookie-settings" class="cookie-settings js-tracked ot-sdk-show-settings" data-tracked-action="Cookie Settings" data-om-config="footer,tk_ca_priva...
Code snapshots of some successful elements
<button type="button" id="domains-button" aria-controls="domains-submenu" aria-expanded="false" aria-haspopup="true" aria-label="Show submenu DOMAINS" aria-labelledby="domains" class="submenu-btn au-domains-button " data-element-location="nav" data-e...
<button type="button" id="website-builder-button" aria-controls="website-builder-submenu" aria-expanded="false" aria-haspopup="true" aria-label="Show submenu WEBSITE BUILDER" aria-labelledby="website-builder" class="submenu-btn au-website-builder-but...
<button type="button" id="email-button" aria-controls="email-submenu" aria-expanded="false" aria-haspopup="true" aria-label="Show submenu EMAIL &amp; MARKETING" aria-labelledby="email" class="submenu-btn au-email-button " data-element-location="nav" ...
<button type="button" id="hosting-button" aria-controls="hosting-submenu" aria-expanded="false" aria-haspopup="true" aria-label="Show submenu WEB HOSTING" aria-labelledby="hosting" class="submenu-btn au-hosting-button " data-element-location="nav" da...
<button type="button" id="security-menu-item-button" aria-controls="security-menu-item-submenu" aria-expanded="false" aria-haspopup="true" aria-label="Show submenu SECURITY" aria-labelledby="security-menu-item" class="submenu-btn au-security-menu-ite...
<button type="button" class="btn au-mobile-chat submenu-btn" data-element-location="nav" data-element-type="link" data-element-label="chat" >Chat</button>
<button type="submit" class="au-mobile-search-btn" data-element-location="nav" data-element-type="button" data-element-label="search" ><svg class="icon search mobile-search" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1792 1792" aria-labelledby="...
<button type="submit" class="btn btn-primary" data-element-location="hero" data-element-type="cta" data-element-label="search" >Search</button>
<button type="submit" id="me-tld-search" class="btn-search" data-element-location="body" data-element-type="button" data-element-label="me-tld-btn" ><svg class="icon search " xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1792 1792" aria-labelledby=...
<button type="submit" id="store-tld-search" class="btn-search" data-element-location="body" data-element-type="button" data-element-label="store-tld-btn" ><svg class="icon search " xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1792 1792" aria-label...
Good ScoreButtons cannot be empty
Good ScoreLinks cannot be empty
Bad ScoreLinks that open in new tabs/windows should be tagged for assistive technology
10
Total Elements
1
Successes
9
Failures
Requirement: Links that open in a new tab or a new window should either have an "aria-label" attribute or a screen-reader only element explaining to screen-readers that this opens in a new tab.

Code snapshots of some failed elements
<a href="/help" target="_blank" rel="noopener noreferrer" data-element-location="nav" data-element-type="link" data-element-label="knowledge-base" class="au-mobile-knowledge-base submenu-btn" >Knowledge Base</a>
<a href="https://newfold.com/privacy-center" data-element-location="footer" data-element-type="link" data-element-label="privacy-notice" target="_blank" >Privacy Notice</a>
<a href="https://www.newfold.com/privacy-center/addendum-for-california-users" data-element-location="footer" data-element-type="link" data-element-label="do-not-sell-my-info" target="_blank" rel="noopener noreferrer" >Do Not Sell My Personal Informa...
<a href="https://www.facebook.com/Domaindotcom/" data-element-location="footer" data-element-type="link" data-element-label="facebook" target="_blank" rel="noopener noreferrer" class="social-links" ><img src="/static/cs/img/icons/socialFacebook.svg" ...
<a href="https://twitter.com/domaindotcom" data-element-location="footer" data-element-type="link" data-element-label="twitter" target="_blank" rel="noopener noreferrer" class="social-links" ><img src="/static/cs/img/icons/socialTwitter.svg" alt="twi...
<a href="https://www.instagram.com/domaindotcom/" data-element-location="footer" data-element-type="link" data-element-label="instagram" target="_blank" rel="noopener noreferrer" class="social-links" ><img src="/static/cs/img/icons/socialInstagram.sv...
<a href="https://www.linkedin.com/company/domaindotcom/" data-element-location="footer" data-element-type="link" data-element-label="linkedin" target="_blank" rel="noopener noreferrer" class="social-links" ><img src="/static/cs/img/icons/socialLinked...
<a href="https://www.youtube.com/channel/UCn7-1xgGV6AfaqcSBLYdGyw" data-element-location="footer" data-element-type="link" data-element-label="youtube" target="_blank" rel="noopener noreferrer" class="social-links" ><img src="/static/cs/img/icons/soc...
<a href="https://open.spotify.com/user/w51o7xrhasfl2zur0ql1zvks7?si=13bcebdb475b40a0" data-element-location="footer" data-element-type="link" data-element-label="spotify" target="_blank" rel="noopener noreferrer" class="social-links" ><img src="/stat...
Code snapshots of some successful elements
<a href="/help/article/hosting-and-domain-renewal-pricing" data-element-location="body" data-element-type="link" data-element-label="renew" target="_blank" >renew at the regular rate.</a>
Titles
SCORE: 75
Good ScoreEvery page should include a single H1 title
Requirement: An H1 title provides information to blind-users using screen-readers of what the main topic of the page is and each page should have exactly one H1 title.

Code snapshots of some successful elements
<h1 class="hero-header " >Do more <div class="tlds"><span class="red-dot">.</span><div class="slidingVertical"><span>online</span><span>travel</span><span>tech</span><span>fun</span></div></div></h1>
Good ScoreTitles cannot be empty
28
Total Elements
28
Successes
0
Failures
Requirement: HTML title elements (H1-6) should have texts. If images or links are used, they should include an alternative or screen-reader only text.

Bad ScoreTitles built as text tags should be labeled as headings for assistive technology
Good ScoreTitles should have a consistent hierarchy
Requirement: Title levels should not be skipped and should be built with consistent hierarchy. For example: you cannot have "h4" titles and "h2" titles without having "h3" titles.

Orientation
SCORE: 33
Good ScoreInteractive elements should be navigable using the keyboard
Requirement: Interactive elements such as links, buttons and form fields should all be navigable using the keyboard by either using a focusable element (a, button, input, etc.) or including the "tabindex" attribute that equals to "0".

Bad ScoreKeyboard focus should have a noticeable outline
Active popups should be tagged for assistive technology
Requirement: Active popups should include an "aria-modal" attribute that equals to "true", and a "role" attribute that equals to "dialog", so screen-reader users know how to navigate within it.

Neutral Score: these elements don't exist or irrelevant to that site rendering the requirement non-applicable.
Bad ScoreEvery page should include hidden links that allow skipping blocks
Manipulatively hidden interactive elements should be excluded from assistive technology
Manipulatively hidden interactive elements should be excluded from tabindex
Menus
SCORE: 100
Good ScoreMenus should be tagged for assistive technology
Menu dropdowns should be tagged for assistive technology
Dropdown expanded/collapsed state should be represented in the code
Graphics
SCORE: 92
Good ScoreObjects and embedded text of images should be described for assistive technology
13
Total Elements
13
Successes
0
Failures
Requirement: Images should have an alternative text description that describes both the objects and the embedded text that the image contains, using the "alt" attribute.

Code snapshots of some successful elements
<img alt="Domain.com Logo" src="/static/cs/img/logo.svg" height="42">
<img srcset="/static/cs/img/pages/home/professional-email-card-s.png" alt="man checking email" width="290px" height="100%">
<img srcset="/static/cs/img/pages/home/security-card-s.png" alt="woman talking on phone" width="290px" height="100%">
<img srcset="/static/cs/img/pages/home/card-header-graphic01-s.png" alt="design graphic" width="290px" height="40px">
<img srcset="/static/cs/img/pages/home/card-header-graphic02-s.png" alt="design graphic" width="290px" height="40px">
<img srcset="/static/cs/img/pages/home/card-header-graphic03-s.png" alt="design graphic" width="290px" height="40px">
<img src="/static/cs/img/inverted-logo.svg" alt="logo" width="252" height="254">
<img src="/static/cs/img/icons/socialFacebook.svg" alt="facebook" width="11" height="23">
<img src="/static/cs/img/icons/socialTwitter.svg" alt="twitter" width="22" height="19">
<img src="/static/cs/img/icons/socialInstagram.svg" alt="Instagram" width="22" height="22">
Background images behaving as standard images should be tagged and described
Mediocre ScoreNon-functional icons/spacers should be excluded from assistive technology
Figure elements should have text or be excluded from assistive technology
Image map areas should be described for assistive technology
Tracker images (pixels) should be excluded from assistive technology
Forms
SCORE: 42
Bad ScoreForm fields should be properly labeled
Form fields should not have duplicate ID selectors
Required form fields should be tagged for assistive technology
Validation status of form fields should be represented in the code
Mediocre ScoreIncorrect use of aria-describedby/labeledby
Good ScoreAll forms have associated submission buttons
Bad ScoreSearch forms should be tagged for assistive technology
Document
SCORE: 92
Good ScoreThe HEAD element should include a title element with the name of the page
Good ScoreThe HTML element should include a proper "lang" attribute
Good ScoreMeta viewport allows display scaling of at least 200%
Mediocre ScorePage landmarks should be tagged and described for assistive technology
Readability
SCORE: 92
Good ScoreFont sizes should be large enough to be readable
Good ScoreLetter spacing should be wide enough to be readable
Mediocre ScoreForeground and background colors have sufficient contrast
Carousels
NEUTRAL
Carousels should be labeled accordingly and be presented as landmarks
Next and previous buttons should be labeled accordingly
Requirement: Control elements such as next and previous icons should include an "aria-label" attribute or a screen-reader only text that explains the functionality to screen-readers.

Neutral Score: these elements don't exist or irrelevant to that site rendering the requirement non-applicable.
Carousels should not be tagged as live regions for assistive technology
Requirement: A common accessibility mistake is to treat carousels as live regions using the "aria-live" attribute and the "polite" or the "assertive values. Using this practice will cause screen-readers to cut and skip announcements.

Neutral Score: these elements don't exist or irrelevant to that site rendering the requirement non-applicable.
Carousel pagination buttons should be labeled accordingly
Requirement: Carousel pagination items (usually used as small dots at the bottom of the carousel) should indicate their functionality and slide number they control using an "aria-label" and a screen-reader only text.

Neutral Score: these elements don't exist or irrelevant to that site rendering the requirement non-applicable.
Tables
NEUTRAL
Tables used for building layouts should be tagged as presentation
Requirement: Tables that are being used as layout structures should include a "role" attribute that equals to "presentation" to indicate to screen-readers that it is not really a table.

Neutral Score: these elements don't exist or irrelevant to that site rendering the requirement non-applicable.
Avoid nesting tables or exclude their role from assistive technology
Requirement: If nested tables are used (a table element within a table cell), set the nested tables "role" attribute to "presentation".

Neutral Score: these elements don't exist or irrelevant to that site rendering the requirement non-applicable.
Headless tables should have fallback rows for assistive technology
General
NEUTRAL
Deleted text should include an additional description
Breadcrumbs should be labeled accordingly and tagged as navigation
Requirement: Breadcrumbs main element should include a "role" attribute that equals to "navigation" and an "aria-label" attribute that describes the functionality."

Neutral Score: these elements don't exist or irrelevant to that site rendering the requirement non-applicable.
Reviews and ratings should be labeled and described accordingly
Include a web accessibility interface
Iframe elements should be labeled or titled
accessScan is powered by
￼
￼￼

Results are ready!
